src/renderer/src/components/TopBar.vue

```diff
@@ -263,4 +316,8 @@ const handleStopMoving = () => {
 .icon-active-red .icon {
   @apply text-red-500;
 }
+
+.save-button {
+  @apply text-red-500;
```

Comment:
ill follow whatever you want. i wanted red because it is not saved until the user clicks it. sort of like when you just go into settings and change rest hps to 30.

this might resolve itself by later just checking when exiting or switching profiles by showing a prompt to save